.github/workflows/integration.yml

```diff
@@ -24,6 +24,11 @@ jobs:
           sed -i -Ee's#git@github.com:([^/]*)/(.*).git#https://github.com/\1/\2.git#' .gitmodules
           git submodule init
           git submodule update
+          # TEMP: use the branch with the fix
```

Comment:
Still necessary?

Reply:
The fix is now merged, but doesn't fix to address the issue. In any case, I removed that change.